CityPuzzle/Game Room/CreateGamePage.xaml.cs

```diff
@@ -11,40 +13,47 @@ namespace CityPuzzle
     [XamlCompilation(XamlCompilationOptions.Compile)]
     public partial class CreateGamePage : ContentPage
     {
-        public static Room NewRoom;
+        public static Lazy<Room> NewRoom= new Lazy<Room>();
         public List<Lazy<Puzzle>> DefaultPuzzles;
```

Comment:
Default values should be initiaded somehow. If there is no default value here change name